webapp/src/locales/de.json

```
@@ -1019,9 +1019,9 @@
         "Limits": "Limits",
         "BatterySoCLimits": "Batterie SoC-Limits",
         "VoltageLimit": "Ladespannungslimit",
-        "enableVoltageLimit": "Start Spannungslimit",
-        "stopVoltageLimitHint": "Maximal Spannung des Ladegeräts. Entspricht der geünschten Ladeschlussspannung der Batterie. Verwendet für die Automatische Leistungssteuerung und beim Notfallladen",
-        "enableVoltageLimitHint": "Die automatische Leistungssteuerung wird deaktiviert wenn die Ausgangsspannung über diesem Wert liegt und wenn gleichzeitig die Ausgangsleistung unter die minimale Leistung fällt.\nDie automatische Leistungssteuerung wird re-aktiveiert wenn die Batteriespannung unter diesen Wert fällt.",
+        "enableVoltageLimit": "Start-Spannungslimit",
+        "stopVoltageLimitHint": "Maximal Spannung des Ladegeräts. Entspricht der gewünschten Ladeschlussspannung der Batterie. Verwendet für die Automatische Leistungssteuerung und beim Notfallladen",
+        "enableVoltageLimitHint": "Die automatische Leistungssteuerung wird deaktiviert wenn die Ausgangsspannung über diesem Wert liegt und wenn gleichzeitig die Ausgangsleistung unter die minimale Leistung fällt.\nDie automatische Leistungssteuerung wird reaktiveiert wenn die Batteriespannung unter diesen Wert fällt.",
```

Hier hast du einen typo übernommen.

```suggestion
        "enableVoltageLimitHint": "Die automatische Leistungssteuerung wird deaktiviert wenn die Ausgangsspannung über diesem Wert liegt und wenn gleichzeitig die Ausgangsleistung unter die minimale Leistung fällt.\nDie automatische Leistungssteuerung wird reaktiviert wenn die Batteriespannung unter diesen Wert fällt.",
```